src/Tribe/Views/V2/Template/Excerpt.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Handles the manipulation of the excerpt.
+ *
+ * @since   4.9.10
+ *
+ * @package Tribe\Events\Views\V2\Template
+ */
+
+namespace Tribe\Events\Views\V2\Template;
+use Tribe__Template as Base_Template;
```

Comment:
:no_entry_sign: There must be one blank line after the namespace declaration

<sub>hash: 630798932d622b0a555c7b09e2bd10c8</sub>